daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Error.scala

```diff
@@ -9,33 +9,41 @@ import com.daml.lf.language.Ast
 import com.daml.lf.transaction.{GlobalKey, NodeId}
 import com.daml.lf.value.Value
 
-sealed abstract class Error {
-  def msg: String
+sealed trait Error {
+  def message: String
 }
 
 object Error {
 
+  final case class Internal(
```

Comment:
I assume the reasoning for having only a single internal error is that callers shouldn’t handle any of those?

Reply:
Yes. 
Furthermore, I included the name of the function where the error is thrown, this makes the classification of the different `InternalError`s a bit redundant.